docs/faq-build.md

```diff
@@ -8,20 +8,56 @@ The development process uses *two web servers* running in tandem:
 * A "front-end" web server which hosts your Fable application. The SAFE template uses the [Webpack Dev Server](https://github.com/webpack/webpack-dev-server) for this.
 
-![](img/faq-build-1.png)
+```mermaid
+flowchart LR
+subgraph c[localhost:8080]
+js>Fable-compiled JS]
+webpack(Webpack dev server)
+js -- hot module replacement --- webpack
+end
+subgraph s[localhost:8085]
+dotnet(dotnet watch run)
+saturn(Saturn on Kestrel)
+saturn --- dotnet
+end
+c -- /api redirect --> s
+```
+
 
 The backend build makes use of `dotnet watch` to allow you to make changes to your server-side application and have the server automatically restart with the latest changes. Since your back-end applications will typically be stateless, this permits a rapid development workflow.
 
 On the client side, your code is compiled using webpack. This provides many benefits for working with javascript, such as minification and bundling, as well as [hot module replacement](feature-hmr.md), a feature made possible when using the webpack dev server. This means that you can continually make changes to your client application code and can rapidly see the results reflected in your browser, without the need to fully reload the application.
 
 ### Webpack Dev Server
-It's important to note that the webpack dev server is configured to automatically route traffic intended for `api/*` routes to the back-end web server. This *simulates* how a SAFE application might work in a production environment, with both client and server assets served from a single web server. This also allows you to not worry about ports and hosts for your backend server in your client code, or [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues. 
+It's important to note that the webpack dev server is configured to automatically route traffic intended for `api/*` routes to the back-end web server. This *simulates* how a SAFE application might work in a production environment, with both client and server assets served from a single web server. This also allows you to not worry about ports and hosts for your backend server in your client code, or [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues.
 
 ## Running SAFE applications in production
 In a production environment, you won't typically use the webpack dev server. Instead, it's more common to use webpack as a one-off compiler step to create your bundled javascript from your Fable app (plus dependencies), and then deploy this along with your back-end web server which also hosts that content directly. For example, you can use Saturn to host the static content required by the application e.g. HTML, JS and CSS files etc. as well as your back-end APIs. This fits very well with standard CI / CD processes, as a build step in your FAKE script or VSTS / AppVeyor / Travis step etc.
 
+```mermaid
```

Comment:
Maybe worth showing that `myapp.dll` is also loaded as part of running in production?